code/mob/living/critter/big_animal.dm

```diff
@@ -41,4 +52,22 @@
 		HH.icon_state = "mouth"					// the icon state of the hand UI background
 		HH.name = "mouth"						// designation of the hand - purely for show
 		HH.limb_name = "teeth"					// name for the dummy holder
-		HH.can_hold_items = 0
+		HH.can_hold_items = FALSE
+
+	critter_attack(var/mob/target)
+		var/datum/targetable/critter/bite = src.abilityHolder.getAbility(/datum/targetable/critter/bite/big)
+		var/datum/targetable/critter/slam = src.abilityHolder.getAbility(/datum/targetable/critter/slam)
+		if (!slam.disabled && slam.cooldowncheck())
+			slam.handleCast(target)
+		if (!bite.disabled && bite.cooldowncheck() && prob(40))
+			bite.handleCast(target)
```

Comment:
feels like this should be an `else if`, otherwise it's possible to get slammed and bitten in the same tick

Reply:
That is intentional. Its like the lion is lunging at you with it fangs/paws. can be changed if you wish.